### PR TITLE
[Task #68005]: Login/ Logout

### PR DESCRIPTION
**Pull request by BuiMinhDung711**
## Related Tickets:
- [#68005](https://edu-redmine.sun-asterisk.vn/issues/68005)

## WHAT this PR does?
- Set up login logout feature: update migration, config view, seed admin account and role

## Notes Impacted Areas:
- ```app/Models```
- ```database/seeders```
- ```database/migrations```

## Attachments:
- image
- image

### CODE REVIEW

**Comment by hoangnbvn** [comment]
ok

**Comment by BuiMinhDung711** [comment]
ready

File: database/seeders/CreateInitialAdminAccount.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        User::create([
+            'email' => 'bui.minh.dung@sun-asterisk.com',
+            'password' => bcrypt('Admin123.'),
+            'first_name' => 'Administrator',
+            'last_name' => 'Account',
+            'is_active' => true,
+            'username' => 'administrator@account',
+            'phone' => '0986653341',
+            'email_verified_at' => now(),
+            'address' => 'Developer',
```
Comment (ghost) [inline database/seeders/CreateInitialAdminAccount.php]: address = developer?

File: database/seeders/CreateInitialAdminAccount.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        // unguard(): disable all mass assignable restrictions.
+        User::unguard();
+
+        User::create([
+            'email' => 'bui.minh.dung@sun-asterisk.com',
+            'password' => bcrypt('Admin123.'),
+            'first_name' => 'Administrator',
+            'last_name' => 'Account',
+            'is_active' => true,
+            'username' => 'administrator@account',
```
Comment (ghost) [inline database/seeders/CreateInitialAdminAccount.php]: username có @?

File: database/seeders/CreateInitialAdminAccount.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            'phone' => '0986653341',
+            'email_verified_at' => now(),
+            'address' => 'Developer',
+        ]);
+
+        $user = User::find(1);
```
Comment (ghost) [inline database/seeders/CreateInitialAdminAccount.php]:
```suggestion
        $user = User::where('email', ...)->first();
```

nếu query tạo user ko phải id = 1 thì sao?

File: database/seeders/CreateInitialAdminAccount.php [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        $user->roles()->attach(2);
```
Comment (ghost) [inline database/seeders/CreateInitialAdminAccount.php]: nếu role tạo bên dưới ko phải 2 thì sao?

**Comment by BuiMinhDung711** [comment]
ready

**Pull request merged**